errtracker/errtracker_test.go

```diff
@@ -35,14 +35,15 @@ import (
 
 	"github.com/snapcore/snapd/arch"
 	"github.com/snapcore/snapd/errtracker"
+	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/release"
 )
 
 // Hook up check.v1 into the "go test" runner
 func Test(t *testing.T) { TestingT(t) }
 
 type ErrtrackerTestSuite struct {
-	restorer func()
+	restorer []func()
```

Comment:
nitpick: we have that test case that does this kind of thing automatically, not sure if you want to use it but I wanted to at leat mention it